### ISSUE TITLE
pretrained model is very important！

### ISSUE THREAD

**Issue by AaronLeong** [huaifeng1993/DFANet#2]
I try to reproduce this paper
I have got  0.88 pixel accuracy(trainset),  but only got 0.53 miou(valset)， So I think the pretrained model is very important！ you can try to use another pretrained backbone.

**Comment by huaifeng1993** [comment]
So do i !.

**Comment by lizhengwei1992** [comment]
@huaifeng1993 对于模型的预训练，何恺明等人前段时间的一篇论文Rethinking ImageNet Pre-training里实验说不使用imagenet上预训练的模型的backbone也可以达到相同的精度，从你目前的训练来看，这种说法可能是有问题的吗？

**Comment by huaifeng1993** [comment]
@lizhengwei1992 在我训练结果上来看，使用预训练模型的情况下，加速了收敛。对最终的结果提升不是很高。但据作者所所说，使用预训练模型的情况下，训练结果有10个点左右的提升。我和@sonack [问题1](https://github.com/huaifeng1993/DFANet/issues/1)交流了一下，我看了他的训练曲线，使用预训练模型对结果影响还是比较大的。在我贴的readme中的训练曲线图中，橙色线是我没有使用预训练模型跑的结果，蓝色线是使用了在imagenet-1k训练了22个epoch的预训练模型的结果。另外两条线，是调整了最大迭代次数（等于增大了学习率）的训练结果。受限于实验条件（只有一块2080），图像输入的大小和，batchsize都经过了缩水，我所做的实验并不充分，所以无法给出更多的意见。

**Comment by Serge-weihao** [comment]
@huaifeng1993 你对论文里面1024X1024还有512X1024的crop size做测试是怎样理解的，是把图像re-scale到1024X1024或者512X1024，还是裁剪成两份或者四份
